includes/enrolment/class-sensei-course-enrolment.php

```diff
@@ -584,6 +584,17 @@ public function withdraw( $user_id ) {
 	 * @return boolean If user is enrolled.
 	 */
 	public function enrol( $user_id ) {
+
+		/**
+		 * Notify when a user should be enrolled to the course.
+		 *
+		 * @since 3.13.2
+		 *
+		 * @param int $course_id Course that the user will be enrolled to.
+		 * @param int $user_id   User ID.
+		 */
+		do_action( 'sensei_enrol_user', $this->course_id, $user_id );
```

Comment:
I think this name can be a little confusing for an action. Because the people can understand that it's for any type of enrol.
I'm thinking something like `sensei_admin_enrol_user `. WDYT?